src/Jackalope/Version/VersionHistory.php

```diff
@@ -178,9 +188,11 @@ public function getVersionByLabel($label)
      */
     public function addVersionLabel($versionName, $label, $moveLabel)
     {
-        // @codeCoverageIgnoreStart
-        throw new NotImplementedException();
-        // @codeCoverageIgnoreEnd
+        $version = $this->getVersion($versionName);
+        $path = $version->getPath();
+
+        $this->objectManager->addLabel($path, $label, $moveLabel);
+        $this->versionLabels[$label] = $version;
```

Comment:
Thats right. So I added an init method, that's called in the addVersionLabel and in getVersionLabels